lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -39,6 +39,27 @@ def self.run_cmd_with_bolt(cmd, nodes)
       return success
     end
 
+    # Run a task with bolt on given nodes
+    #
+    # @author Sam Woods
+    #
+    # @param [string] task Task to run on nodes
+    # @param [array] env_vars env_vars to set
```

Comment:
Since we are passing them as params to the task, we should probably rename them from env_vars to something like task_params

Reply:
Also, I would actually like to bring the params in as a hash.  Down the line when they get to be more than a few we are going to want to convert to generating a json file I think.  Having it come in as a hash means we can change the implementation later without affecting the interface.

Reply:
Filed SLV-175 to cover converting these in the future as tech debt.